deps/build.jl

```diff
@@ -1,15 +1,61 @@
-using Libdl
-
-using BinDeps
-# If I do not do using CxxWrap, when BinDeps try dlopen on libsdpawrap.so, I get
-# ERROR: could not load library "~/.julia/v0.5/SDPA/deps/usr/lib/libsdpawrap"
-# libcxx_wrap.so.0: cannot open shared object file: No such file or directory
+using BinaryProvider # requires BinaryProvider 0.3.0 or later
```

Comment:
In REQUIRE, it says that it needs BP v0.5.3 or later

Reply:
updated